### PR TITLE
Fixed: #46: Added Border & Reduced the space between down arrow icon & option text

### PR DESCRIPTION
**Pull request by chinmaym07**
fixes: #46 

https://user-images.githubusercontent.com/44670961/154821094-8c589d70-ce8f-4f41-ab1c-74a40c53687b.mp4

Signed-off-by: chinmaym07 <b418020@iiit-bh.ac.in>

### CODE REVIEW

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]
The button appears to be very different from the secondary button. Checkout the `buttons.scss` file for the exact styling, or try to reuse the `Button.js` component

File: src/assets/scss/components/select.scss [hunk 1]
```diff
@@ -22,10 +22,11 @@ select {
   background: $eos-select-bg-active;
   background-color: transparent;
   background-size: $eos-select-bg-size;
-  border: 0;
-  border-radius: 0;
+  border: 1px solid $eos-form-element-color;
```
Comment (cyntss) [inline src/assets/scss/components/select.scss:25]: Use the same color for the border as used in the buttons. I think the $secondary variable is the right one

**Review by cyntss — CHANGES_REQUESTED** [review]

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]
Great work @chinmaym07
The border now looks similar to the Secondary Button. Can you try to bring the text a little lower (Same as the `Download EOS Icons` button) and make the text similar to the Secondary Button
opera_wBUipTyfIX
opera_iTP0ZTblWT

**Review by vinayaksh42 — APPROVED** [review]
LGTM! @chinmaym07

**Pull request merged**